README.md

```diff
@@ -18,4 +18,4 @@ spin watch # Uses spin watch to run the website and reload when content changes
 
 Editing the contents:
 
-If you would like to edit the contents of the [developer.fermyon.com](https://developer.fermyon.com/) website, and contribute your changes, please visit the [Contributing to Docs](https://developer.fermyon.com/common/contributing-docs) page.
+If you would like to edit the contents of the [developer.fermyon.com](https://developer.fermyon.com/) website, and contribute your changes, please visit the [Contributing to Docs](spin/contributing-docs) page.
```

Comment:
This link doesn't work for me because of this being the readme rather than on the site itself - might need to be a GH path or an absolute link to `developer.fermyon.com`

Reply:
Thanks Ivan, tested this in a fork and the best way to get this to resolve properly is just use the full URL.